electronics/ohms_law.py

```diff
@@ -0,0 +1,46 @@
+def ohms_law(voltage: float, current: float, resistance: float) -> float:
+    """
+    This function apply ohm's law, on any two given electrical values,
+    which can be voltage current resistance,
+    and then return name and value pair in the form of dictionary
+    >>> ohms_law(voltage=10, resistance=5, current=0)
+    {'current': 2.0}
+    >>> ohms_law(voltage=0, current=0, resistance=10)
+    Traceback (most recent call last):
+      File "<stdin>", line 14, in <module>
+    ValueError: Only one argument can be 0 at the time
+    >>> ohms_law(resistance=0, voltage=-10, current=1)
+    {'resistance': -10.0}
+    >>> ohms_law(voltage=0, current=-1.5, resistance=2)
+    {'voltage': -3.0}
+    """
+    if voltage == 0:
+        if current == 0 or resistance == 0:
+            raise ValueError("Only one argument can be 0 at the time")
+        elif resistance <= 0:
+            raise ValueError("Resistance can't be 0 or in negative")
+        else:
+            result = {"voltage": float(current * resistance)}
+            return result
```

Comment:
Simply remove variables that are created on one line and deleted on the next line.
```suggestion
            return {"voltage": float(current * resistance)}
```

Reply:
Why do we cast as `float()` on this branch but not on the other two branches?

Reply:
Because if you divide two values its gives you float but not in case for the multiplication , and remember i give return type float in type hint